python/tests/test_util.py

```diff
@@ -540,3 +540,29 @@ def test_length_bad_value(self, length):
     def test_length_bad_type(self, length):
         with pytest.raises(TypeError, match="argument must be a string"):
             tskit.random_nucleotides(length)
+
+
+#
+# Tests for allele_remap
+#
+@pytest.mark.parametrize(
```

Comment:
Rather than have the same code repeated in three tests, combine all the `parametrize` lists, adding comments like "differs in length" and "both empty" before the line of those ones.